### DISCUSSION TITLE
HAA and konnected.io

### DISCUSSION THREAD

**Discussion by adamwhiteland** [RavenSystem/esp-homekit-devices#2038] [Q&A]
Hello, has anyone managed to successfully flash and configure the konnected.io board? 

[https://konnected.io](https://konnected.io)

**Comment by PhaseAngle** [comment]
The GPIO map is provided on their site

https://help.konnected.io/support/solutions/articles/32000026808-zone-to-gpio-pin-mapping

I haven't tried making an alarm system yet. It might be next after I replace the controller of a heat pump hot water system with a Kincony board
